store/postgres/asset_model.go

```diff
@@ -0,0 +1,118 @@
+package postgres
+
+import (
+	"database/sql/driver"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/odpf/columbus/asset"
+)
+
+type Asset struct {
```

Comment:
I propose to standardize the naming of domain and DB model.
Model naming always end with suffix `..Model`
Domain name is the entity name itself.

For example:
The DB model of asset is `AssetModel`
The domain model of asset is `Asset`

To simplify, I think we don't need to explicitly mention that a struct is a `Domain` and we just need to consider every entity packages outside `store` package is our domain codes. Therefore, the function helper to convert DB model to domain could be more explicit. Instead of having a function called `toDomain` or  `toAssetDomain`, we could just have an explicit function `toAsset` where we already know that `Asset` is a domain.

wdyt @StewartJingga ?

Reply:
Yeah agree with that, never in fond of using domain anyway.

Will apply the changes.

Reply:
@StewartJingga should we update it to `AssetModel` stew?

Reply:
Oh sorry missed this, updating